src/main/kotlin/no/skatteetaten/aurora/boober/service/vault/VaultService.kt

```diff
@@ -108,11 +111,20 @@ class VaultService(
     }
 
     fun deleteVault(vaultCollectionName: String, vaultName: String) {
-        return withVaultCollectionAndRepoForUpdate(vaultCollectionName) { vaultCollection, repo ->
-            findVaultByNameIfAllowed(vaultCollection, vaultName) ?: return@withVaultCollectionAndRepoForUpdate
+        if (vaultName.isBlank()) {
+            throw IllegalArgumentException("vault name can not be empty")
+        }
 
-            vaultCollection.deleteVault(vaultName)
-            gitService.commitAndPushChanges(repo)
+        return withVaultCollectionAndRepoForUpdate(vaultCollectionName) { vaultCollection, repo ->
+            val vault =
+                findVaultByNameIfAllowed(vaultCollection, vaultName) ?: return@withVaultCollectionAndRepoForUpdate
+
+            vault.vaultFolder.deleteRecursively()
```

Comment:
er denne unødvendig siden du bruker git rm?

Reply:
Var mer for lesbarhet, da må vi kanskje lage en ny metode i GitService som indikerer at man også skal slette., fremfor å bruke commitAndPushChanges for sletting også. Er mulig å konfigurere rm-commandoen til å bare legge til endringer til staged. http://archive.eclipse.org/jgit/docs/jgit-2.3.1.201302201838-r/apidocs/org/eclipse/jgit/api/RmCommand.html#setCached(boolean) Kanskje vi bør gjøre det?

Reply:
Jeg ser den når jeg leser den på nytt. Det ville sett veldig rart uten den. Jeg er usikker hva jeg foretrekker. Så har egentlig ingen innvendinger her, du kan bestemme selv om du gidder å gjøre noe med det.